docs/api/browser-window-proxy.md

```diff
@@ -50,4 +50,4 @@ The `BrowserWindowProxy` object has the following instance properties:
 
 #### `win.closed`
 
-A Boolean that is set to true after the child window gets closed.
+A `Boolean` that is set to true after the child window gets closed.
```

Comment:
> optional

This can be parsed out of the description as linked above and I have a branch somewhere that made it API, the problem is not all things are documented as optional correctly in electron.  See

> array yes/no (I can parse that out of the type)

We use standard type array notation `[]` to indicate if it is of type X or an array or type X.  E.g. `X` vs `X[]`

> newlines, lists and programlistings

These mostly get stripped out due to them being HTML in our parser, could totally put them back in 👍 

> the array information isn't extracted out of the union type. 

Thanks for spotting that, raised over at https://github.com/electron/electron-docs-linter/issues/84

> (Also the docs aren't clear whether an array of only one of the types or a mixed-type array can be passed)

Array of any type = `(Foo | Bar)[]`
Any array of single type = `(Foo[] | Bar[])`

Refs:
https://github.com/electron/electron-docs-linter/issues/9
https://github.com/electron/electron-docs-linter/issues/13